client/views/blocks/ModalBlock.js

```diff
@@ -59,6 +59,12 @@ function ModalBlock({ view, errors, appId, onSubmit, onClose, onCancel }) {
 		(event) => {
 			if (event.keyCode === 13) {
 				// ENTER
+				if (
+					event.srcElement &&
+					(event.srcElement.nodeName === 'TEXTAREA' || event.srcElement.localName === 'textarea')
```

Comment:
`srcElement` have some compatibility problems with firefox, how about change to `currentTarget`?

Reply:
I couldn't find `currentTarget` prop attached to this event. The closest I found to this was `target` prop, so I've it to use that instead. Let me know if that should work or not? Thanks!!

Reply:
perfect!